src/CustomerProfile.js

```diff
@@ -0,0 +1,26 @@
+
+import React, { useEffect, useState } from "react";
+const CustomerProfile = props => {
+  const [profile, setProfile] = useState(null);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${props}`)
+      .then(response => {
+        return response.json();
+      })
+      .then(data => {
+        setProfile(data);
+        console.log(props);
+      });
+  }, []);
```

Comment:
This is good, good job - as an improvement notice how this `useEffect` only exists to populate `profile` with data? You could extract both this `useState` and `useEffect` to a custom hook (this is far beyond what is expected)